primitives/src/market.rs

```diff
@@ -182,10 +179,7 @@ impl<AI, BA, BN, M, A> Market<AI, BA, BN, M, A> {
     ) -> Option<MarketAssetClass<MI>> {
         match outcome_report {
             OutcomeReport::Categorical(idx) => match self.scoring_rule {
-                ScoringRule::Orderbook => {
-                    Some(MarketAssetClass::<MI>::CategoricalOutcome(market_id, *idx))
-                }
-                ScoringRule::Lmsr => {
+                ScoringRule::AmmCdaHybrid => {
```

Comment:
It will be impossible to use orderbook / lmsr only?

Reply:
Both just use the scoring rule AmmCdaHybrid.